### ISSUE TITLE
Clarify Guide VS Reference

### ISSUE THREAD

**Issue by scottopell** [Purdue-CSUSB/CSToolsCourse#4]
Now that I'm looking back at the vim reference that I wrote, it's kinda a blend between a guide and a reference.

What would you say the key differences are and how do I make it into one or the other?

Or should these two concepts be combined?

**Comment by nickysemenza** [comment]
@scottopell in my opinion:
guide = more of a tutorial on getting started with the basics with explanations of how/why things work
reference  = you know the basics but maybe you are still new and you forgot one two shortcuts and you can just go look it up on a table, you don't really need any 'explanations' perse 

i feel like /reference/vim.md is a bit of a fusion between the two, but i feel like it is still helpful, because to be honest, once people get a solid grasp they might just go google their vim question instead of looking in this repo?

**Comment by scottopell** [comment]
Yea, I agree with you on the overall idea. Curious what @leviathan747 wants though, I'm happy to modify it to be either one or keep it as the fusion that it is.

And def agree on the "just gonna google it." Once I have a decent grasp of things I'm not going to think to go to cs190 github and look something up, I'm just gonna google it.

**Comment by leviathan747** [comment]
I like the way you have it. I glanced at your reference sheet for about 5 seconds and i knew how to open, close and save files, move in a file and use insert or command mode. It was nicely bolded so the "reference" parts stood out while still providing helpful explanations. I say keep it the way it is. Typically I think the guides/tutorials should focus on specific tasks such as setting up ssh keys or use cases like a debugging example and the references focus more on the tools and how exactly to use them. And as to the "just gonna google it" point, the idea is if they have these reference sheets during their labs, they may remember "we did this in CS 190 lab but I don't remember exactly how" and they will quickly be able to find it. I personally find myself going back to the references I used the first time I do something. We're just trying to make those more accessible since something in this repo will be easier to find than maybe a specific StackOverflow post.

TL;DR keep it the way it is. I give it an A+
